tests/test_utils.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+test_django-watchman
+------------
+
+Tests for `django-watchman` utils module.
+"""
+
+from __future__ import unicode_literals
+
+import unittest
+
+from watchman.utils import get_checks
+
+
+class TestWatchman(unittest.TestCase):
+
+    def assertListsEqual(self, l1, l2):
```

Comment:
Rename first argument for readability? `l1` is about as hard to read as you can get.